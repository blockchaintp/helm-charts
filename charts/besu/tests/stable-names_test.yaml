suite: besu.test for stable names
templates:
  - besu-statefulset.yaml
  - besu-service.yaml
  - besu-serviceaccount.yaml
  - enode-service.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
  - it: check for stable PVC names
    template: besu-statefulset.yaml
    set:
      besu:
        persistence:
          enabled: true
    asserts:
      - matchSnapshot:
          path: spec.volumeClaimTemplates[0].metadata.name
