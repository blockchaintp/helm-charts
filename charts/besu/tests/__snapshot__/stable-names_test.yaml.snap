check for stable PVC names:
  1: |
    data
check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    test-besu
  2: |
    test-besu
  3: |
    test-besu
  4: |
    test-besu-0
  5: |
    test-besu-1
  6: |
    test-besu-2
  7: |
    test-besu-3
